Code/RDBoost/CMakeLists.txt

```diff
@@ -1,6 +1,17 @@
-rdkit_library(RDBoost Wrap.cpp
-              LINK_LIBRARIES RDGeneral ${PYTHON_LIBRARIES} ${Boost_LIBRARIES})
-
+if("${Py_ENABLE_SHARED}" STREQUAL "1")
+  rdkit_library(RDBoost Wrap.cpp
+                LINK_LIBRARIES RDGeneral ${PYTHON_LIBRARIES} ${Boost_LIBRARIES})
+else()
+    rdkit_library(RDBoost Wrap.cpp
+                LINK_LIBRARIES RDGeneral ${Boost_LIBRARIES})
+    if("${PYTHON_LDSHARED}" STREQUAL "")
+    else()
+       # fixes an apple issue
```

Comment:
if it's really an apple issue, shouldn't there be a test around this to make sure it only happens on the Mac?

Reply:
It isn't really an apple issue, it can happen on linux as well, it's just that the compiler suites are are testing on are a bit more lenient.